libraries/aws_cloudfront_distribution.rb

```diff
@@ -68,6 +68,17 @@ def initialize(opts = {})
       end
     end
     @custom_origin_ssl_protocols = @custom_origin_ssl_protocols.uniq.sort
+
+    # Find aws cloudfront distribution origin path.
+    # Either return path string, or ""
+    return unless opts[:origin_domain_name]
+    @cf_origin_domain_name = opts[:origin_domain_name]
```

Comment:
`cf_origin_domain_name` should this be `origin_domain_name` ? because you have defined attr reader as `origin_domain_name`

Reply:
Yes indeed , it was not required, just updated with proper variable